dev/unix/boot-install.sh

```diff
@@ -1,76 +1,76 @@
 #!/usr/bin/env bash
 
-# This is the bootstrap Unix installer served by `https://get.notionjs.com`.
+# This is the bootstrap Unix installer served by `https://get.volta.sh`.
 # Its responsibility is to query the system to determine what OS (and in the
 # case of Linux, what OpenSSL version) the system has, and then proceed to
-# fetch and install the appropriate build of Notion.
+# fetch and install the appropriate build of Volta.
 
-notion_get_latest_release() {
-  curl --silent https://www.notionjs.com/latest-version
+volta_get_latest_release() {
+  curl --silent https://www.volta.sh/latest-version
 }
 
-notion_eprintf() {
+volta_eprintf() {
   command printf "$1\n" 1>&2
 }
 
-notion_info() {
+volta_info() {
   local ACTION
   local DETAILS
   ACTION="$1"
   DETAILS="$2"
   command printf '\033[1;32m%12s\033[0m %s\n' "${ACTION}" "${DETAILS}" 1>&2
 }
 
-notion_error() {
+volta_error() {
   command printf '\033[1;31mError\033[0m: ' 1>&2
-  notion_eprintf "$1"
+  volta_eprintf "$1"
 }
 
-notion_warning() {
+volta_warning() {
   command printf '\033[1;33mWarning\033[0m: ' 1>&2
-  notion_eprintf "$1"
-  notion_eprintf ''
+  volta_eprintf "$1"
+  volta_eprintf ''
 }
 
-notion_request() {
+volta_request() {
   command printf "\033[1m$1\033[0m" 1>&2
-  notion_eprintf ''
+  volta_eprintf ''
 }
 
-notion_install_dir() {
-  printf %s "${NOTION_HOME:-"$HOME/.notion"}"
```

Comment:
Should we add logic around migrating/deleting an existing `$HOME/.notion` directory?

Reply:
Was thinking similar. I'll see if I can whip up something to handle it tomorrow.

Reply:
Added handling in e53e27d.